cdk/visit/lambda_code/log_visit/log_visit.py

```diff
@@ -0,0 +1,196 @@
+import json
+import datetime
+from pydoc import cli
+import boto3
+from boto3.dynamodb.conditions import Key
+from botocore.exceptions import ClientError
+import logging
+import traceback
+import sys
+import os
+import re
+
+
+class LogVisitFunction():
+    """
+    This function will be used to wrap the functionality of the lambda
+    so we can more easily test with pytest.
+    """
+
+    def __init__(self, table, client):
+        self.logger = logging.getLogger()
+        self.logger.setLevel(logging.INFO)
+
+        if table is None:
+            # Get the service resource.
+            dynamodb = boto3.resource('dynamodb')
+            # Get the table name.
+            TABLE_NAME = os.environ["TABLE_NAME"]
+            # Get table objects
+            self.visits = dynamodb.Table(TABLE_NAME)
+        else:
+            self.visits = table
+
+        if client is None:
+            AWS_REGION = os.environ['AWS_REGION']
+            self.client = boto3.client('ses', region_name=AWS_REGION)
```

Comment:
can we make this member variable more descriptive? e.g. `ses_client`

Reply:
Done!